examples/configs/swift/SWIFT.conf

```diff
@@ -0,0 +1,117 @@
+# [FILE PATH OPTIONS]
+
+# Path to where the base folder of where the particle information is. For SWIFT
+# outputs, the FoF group information is also contained within the snapshot, so the 
+# paths should be the same.
+SnapshotPath
+HaloPath
+
+# Base name for the snapshot files, e.g. <SnapshotFileBase>_<snap_nr>.hdf5
+SnapshotFileBase
+
+# Base name for the directories where snapshot subfiles are saved, if applicable.
+# For example, <SnashotDirBase>_<snap_nr>/<SnapshotFileBase>_<snap_nr>.<subfile_nr>.hdf5
+# If there is only one file per snapshot, omit this option.
+SnapshotDirBase
+
+# Base directory of where the HBT catalogues will be saved. The HBT catalogue for each snapshot
+# is saved in its own subdirectory, with the number of subfiles matching the number of MPI ranks.
+SubhaloPath
+
+# [FILE IO OPTIONS]
+# This tells HBT+ that the snapshots are SWIFT-like. DO NOT CHANGE!
+SnapshotFormat  swiftsim
+GroupFileFormat swiftsim_particle_index
+
+# The full range of snapshots that the simulation has. If 10 snapshots exist, this would correspond to
+# 0 and 9, respectively.
+MinSnapshotIndex
+MaxSnapshotIndex
+
+# Which subset of snapshots to analyse in an HBT run. For example, if we only want to analyse every 
+# second snapshot of a ten snapshot run: 1 3 5 7 9
+SnapshotIdList
+MaxConcurrentIO 512
+
+# [PARTICLE PROPERTY OPTIONS]
+# For HYDRO runs, whether gas particles split once they become massive, and if these splits should be 
+# taken into account. If enabled, the particle split scripts NEED to be run (see toolbox/swiftsim).
+ParticlesSplit 1
+
+# [UNIT OPTIONS]
```

Comment:
I can't remember, are these still required or can they be omitted?

Reply:
For swift, they are not required as they are loaded from the snapshot directly. Should I remove them?

Note that I just saw that not specifying the units does not make the code crash. However, if they are not read in from the snapshot (e.g. for other snapshot formats), this means that it will use uninitialised values! Might be worth adding a default value of -1 to the units, and make the code complain if they are not set in the config or directly from the snapshot.

Reply:
My preference for the swift config would be to only keep the options required for swift. Maybe keep a comment line explaining which values are read from the snapshots. 

I think a default of -1 would be a good idea!

Reply:
What I have done now for the SWIFT config is to trim it down to a minimum working example. Any additional options will use the default values. ([ddce77a](https://github.com/SWIFTSIM/HBTplus/pull/64/commits/ddce77aa6fc92d20a9d3c22667247e28bb942b9f) and [1aff321](https://github.com/SWIFTSIM/HBTplus/pull/64/commits/1aff32145e1f96e32b4f1fc587c84f4dcf5a768d))